fs/copy.ts

```diff
@@ -137,7 +137,7 @@ async function copySymLink(
 ) {
   await ensureValidCopy(src, dest, options);
   const originSrcFilePath = await Deno.readLink(src);
-  const type = getFileInfoType(await Deno.lstat(src));
+  const type = getFileInfoType(await Deno.lstat(originSrcFilePath));
```

Comment:
I think this only works when `originSrcFilePath` is an absolute path or relative path accidentally points to path accessible from cwd. `originSrcFilePath` could be relative path from link path to target, which could be inaccessible from `cwd`, and in that case this `lstat` call would fail.

I think this should `Deno.stat(src)` instead with fallback to 'file' type in case `Deno.stat(src)` throws NotFound. What do you think?